src/zh/yymanhua/src/eu/kanade/tachiyomi/extension/zh/yymanhua/YYmanhua.kt

```diff
@@ -0,0 +1,157 @@
+package eu.kanade.tachiyomi.extension.zh.yymanhua
+
+import eu.kanade.tachiyomi.network.GET
+import eu.kanade.tachiyomi.source.model.FilterList
+import eu.kanade.tachiyomi.source.model.Page
+import eu.kanade.tachiyomi.source.model.SChapter
+import eu.kanade.tachiyomi.source.model.SManga
+import eu.kanade.tachiyomi.source.online.ParsedHttpSource
+import okhttp3.HttpUrl.Companion.toHttpUrl
+import okhttp3.Request
+import okhttp3.Response
+import org.jsoup.nodes.Document
+import org.jsoup.nodes.Element
+import java.util.concurrent.TimeUnit
+
+class YYmanhua : ParsedHttpSource() {
+
+    override val baseUrl = "https://www.yymanhua.com"
+    override val lang = "zh"
+    override val name = "YY漫画"
+    override val supportsLatest = true
+
+    companion object {
+        val DESC_REGEX = Regex("\\[\\+展开]|\\[-折叠]")
+        val CHAPTER_REGEX = Regex("第(\\d+(?:\\.\\d+)?)[话話]")
+        val NUM_REGEX = Regex("\\d+")
+        val IMG_REGEX = Regex("var pix=\"(.*?)\".*?var pvalue=\\[\"(.*?)\"")
+        val DECODE_REGEX1 = Regex("return p;\\}\\('(.*?)',(\\d+),(\\d+),'(.*?)'")
+        val DECODE_REGEX2 = Regex("\\b\\w+\\b")
+    }
+
+    override fun headersBuilder() = super.headersBuilder()
+        .set("User-Agent", "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/136.0.0.0 Safari/537.36 Edg/136.0.0.0")
```

Comment:
does site not work with mobile or other specific useragents? because hard coding useragent is generally not a good idea